requirements.txt

```diff
@@ -1 +1,2 @@
 ansible>=2.9
+backoff==1.10.0
```

Comment:
do we still need this?

Reply:
It's part of the attempt to handle the import error, will be removed